tests/python/test_schedule_ops.py

```diff
@@ -22,6 +35,29 @@ def is_pre_volta():
     return prop.major < 7
 
 
+# A helper function to test heuristic schedulers with user schedules
+def _apply_scheduler_helper(schedule, selected_heuristic):
+    # Check that only selected heuristic is available as a scheduler
+    available_heuristics = schedule.find_compatible_schedulers()
+    assert available_heuristics == [selected_heuristic]
```

Comment:
I think we need this
```suggestion
    assert set(available_heuristics) == set(selected_heuristic)
```